signer/tests/integration/transaction_coordinator.rs

```diff
@@ -3559,6 +3588,20 @@ async fn process_rejected_withdrawal(is_completed: bool, is_in_mempool: bool, is
             .await
             .unwrap();
 
+        let sighash = BitcoinTxSigHash {
+            txid: bitcoin_txid,
+            prevout_type: model::TxPrevoutType::SignersInput,
+            prevout_txid: donation.txid.into(),
+            prevout_output_index: donation.vout,
+            validation_result: signer::bitcoin::validation::InputValidationResult::Ok,
+            aggregate_key: aggregate_key.into(),
+            is_valid_tx: false,
+            will_sign: false,
+            chain_tip: bitcoin_chain_tip.block_hash,
+            sighash: bitcoin::TapSighash::from_byte_array([1; 32]).into(),
+        };
+        db.write_bitcoin_txs_sighashes(&[sighash]).await.unwrap();
+
         if is_submitted {
```

Comment:
I guess this is no longer useful right now, but we can remove the `is_submitted` case later. Or just keep it.

Reply:
Removed in https://github.com/stacks-network/sbtc/pull/1406/commits/ceea0fba418c81220f59c4086723c9d850cfb91d.